grisha.pletin/SecondHW/pet.rb

```diff
@@ -0,0 +1,87 @@
+class Pet
+  MAX_VALUE = 100
+  PETS = ['cat', 'dog', 'parrot']
```

Comment:
Please freeze constants
https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/MutableConstant
Also there is a shortcut for these kind of arrays which is
`PETS = %w[cat dog parrrot]`